packages/react-components/src/components/VideoTile.tsx

```diff
@@ -138,7 +162,7 @@ const DefaultPlaceholder = (props: DefaultPlaceholderProps): JSX.Element => {
           aria-label={noVideoAvailableAriaLabel ?? ''}
           showOverflowTooltip={false}
         />
-        {participantState && <Text className={mergeStyles(participantStateStyle)}>{participantState}</Text>}
+        {participantState && <Text className={mergeStyles(participantStateStyle)}>{participantStateString}</Text>}
```

Comment:
Do we also maybe want to embed a Announcer using these strings here as well? or does the narrator catch these string changes when its just in a `Text` tag?

Reply:
Not sure. Lets postpone a11y work as complex as announcer till public preview.